lib/argo_cd/response.rb

```diff
@@ -58,9 +58,17 @@ def sync_status
       body['status']['sync']
     end
 
-    def previous_commit; end
+    def author
+      body['author']
+    end
+
+    def date
+      body['date']
+    end
 
-    def current_deploy_info; end
+    def message
+      body['message']
+    end
 
     def token
       body['token']
```

Comment:
Did you want to add instance variables to these methods too? `@<instance_variable> ||= <...>`

Reply:
I added some instance vars for memoization, but now that I think about it, since these methods are only called (maybe) once, I'm not sure that we would see a great caching benefit.